src/app/plugins/modreport.plugin.ts

```diff
@@ -35,17 +35,24 @@ export class ModReportPlugin extends Plugin {
         await this._handleIssueWarning(message, user_handle, description);
       } else if (sub_command === 'ban') {
         await this._handleIssueBan(message, user_handle, description);
+      } else if (sub_command === 'full') {
+        await this._handleFullList(message, user_handle);
       }
     } catch (e) {
       message.reply('Something went wrong. Did you put the username correctly?');
       this.container.loggerService.error(e);
     }
   }
 
-  private _createReport(message: IMessage, user_handle: string, description?: string) {
+  private async _createReport(message: IMessage, user_handle: string, description?: string) {
+    const id = await Moderation.Helpers.resolveUser(this.container.guildService.get(), user_handle);
+    if (!id) {
+      return;
+    }
+
     const rep: Moderation.Report = new Moderation.Report(
       this.container.guildService.get(),
-      user_handle,
+      id,
```

Comment:
this is that patch right? isnt that checked in already?

Reply:
I didnt update the dev branch before making this PR, next time i push it should be gone